src/child-view-container.js

```diff
@@ -84,24 +97,42 @@ _.extend(Container.prototype, {
     }
 
     // delete custom index
-    _.any(this._indexByCustom, function(cid, key) {
+    _.some(this._indexByCustom, _.bind(function(cid, key) {
       if (cid === viewCid) {
         delete this._indexByCustom[key];
         return true;
       }
-    }, this);
+    }, this));
 
     // remove the view from the container
     delete this._views[viewCid];
 
-    // update the length
-    this._updateLength();
     return this;
   },
 
+  // Call a method on every view in the container,
+  // passing parameters to the call method one at a
+  // time, like `function.call`.
+  call(method) {
```

Comment:
call and apply can be removed.  They don't actually do anything you can't do with the built in underscore functions:  https://github.com/marionettejs/backbone.babysitter/issues/57